nautobot/extras/plugins/views.py

```diff
@@ -115,11 +114,9 @@ def get(self, request, format=None):  # pylint: disable=redefined-builtin
         return Response([self._get_plugin_data(apps.get_app_config(plugin)) for plugin in settings.PLUGINS])
 
 
-class PluginsAPIRootView(NautobotAPIVersionMixin, APIView):
-    _ignore_model_permissions = True
-
-    def get_view_name(self):
-        return "Plugins"
+class PluginsAPIRootView(AuthenticatedAPIRootView):
+    name = "Apps"
```

Comment:
Changing this back to plugins, 1.6 was never updated to use "App" everywhere like 2.x was.